calico/maintenance/ebpf/install.md

```diff
@@ -307,10 +306,18 @@ spec:
   # EKS with Bottlerocket as node image only:
   # flexVolumePath: /var/lib/kubelet/plugins
     
-  # Install Calico Enterprise
-  variant: TigeraSecureEnterprise
-  
-  # ... remainder of the Installation resource varies by platform ...
+  # Install Calico Opensource
+  variant: calico
```

Comment:
Is calico supposed to be lower case?  Seems inconsistent?

Reply:
According to the docs yes :confused:  https://github.com/projectcalico/calico/blob/d962031a0fa1ecf1ca7d69bd2cc292cbd22b6e19/calico/reference/installation/_api.html#L203

Reply:
@fasaxc nice catch seems like more fixes are in order
https://github.com/tigera/operator/blob/337fd699162839661b4a8a553b44222ec3c19df8/pkg/crds/operator/operator.tigera.io_installations.yaml#L5879